src/tts.py

```diff
@@ -200,9 +200,15 @@ def merge_audio_files(self, audio_files, voiceline_file_name):
     
 
     @utils.time_it
-    def _synthesize_line(self, line, save_path):
+    def _synthesize_line(self, line, save_path, aggro=0):
+        pluginsContext = {}
+        # in combat
+        if (aggro == 1):
```

Comment:
Would rather have this represent that the character is in combat, not just taunted to combat by the player.